docs/reference/search/search-your-data/semantic-search-elser.asciidoc

```diff
@@ -216,6 +216,69 @@ weights.
 ----
 // NOTCONSOLE
 
+
+[discrete]
+[[text-expansion-compound-query]]
+=== Combining `text_expansion` with a compound query
+
+You can combine `text_expansion` with a <<compound-queries,compound query>> for 
+example with a `query_string` that uses the same query text as the 
+`text_expansion` query. This enables you to combine the search results 
+from both queries. You can increase or decrease the relevance scores of each 
+query by using the `boost` parameter.
```

Comment:
With the next two paragraphs added in https://github.com/elastic/elasticsearch/pull/96460/commits/88118b99421749a5b2f4ba8e579e9fd3b1a650b5, this paragraph and the next have become repetitive. Maybe we drop this paragraph?